src/time_integration/low_storage_integrator.cpp

```diff
@@ -130,6 +130,40 @@ LowStorageIntegrator::LowStorageIntegrator(const std::string &name)
     gam0[2] = 2.0 / 3.0;
     gam1[2] = 1.0 / 3.0;
     c[2] = 0.5;
+  } else if (name_ == "rk34") {
```

Comment:
Might also we worth to add it to the list at the bottom of https://github.com/parthenon-hpc-lab/parthenon/blob/develop/doc/sphinx/src/integrators.rst#lowstorageintegrator